dtbase/models/arima/arima/arima_pipeline.py

```diff
@@ -10,46 +9,42 @@
 from sklearn.model_selection import TimeSeriesSplit
 from statsmodels.tsa.statespace.sarimax import SARIMAX, SARIMAXResultsWrapper
 
-from dtbase.models.utils.config import config
+from dtbase.models.arima.arima.config import ConfigArima
```

Comment:
I don't love have a directory called arima in a directory called arima. Could we just extract everything from the 2nd arima directory?

Reply:
I don't really see the function of it either, can change.